sixtyfps_runtime/corelib/timers.rs

```diff
@@ -360,14 +361,19 @@ pub(crate) mod ffi {
     /// The timer MUST be stopped with sixtyfps_timer_stop
     #[no_mangle]
     pub extern "C" fn sixtyfps_timer_start(
+        id: i64,
```

Comment:
can that be documented that -1 means create?

Reply:
Sure, can do